dips/dip-10.md

```diff
@@ -13,47 +13,57 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+Diem ID provides a convenient method for identifying users within a VASP. For peer-2-peer payments, Diem ID allows users to exchange human-readable identifiers as either the sender or the recipient of peer-to-peer payments. Instead of sending a [dip-5 address](https://github.com/diem/dip/blob/master/dips/dip-5.md) (that ideally needs to be refreshed for each payment), the receiver can just share their Diem ID once. In this case Diem ID plays the role of an email address for payments. Benefits for this use case:
+* Privacy: There are concerns around leakage of personally identifiable information (PII) when using subaddresses. Diem ID provides better privacy compared to sub-addresses, because unlike sub-addresses, Diem ID is intended to be used in tandem with an off-chain protocol. Hence, Diem ID eliminates the need to expose PII on the chain 
+* Persistent Identifiers: Currently there is a lack of persistent user identifiers in the Diem ecosystem. while subaddresses need to be refreshed for each payment, Diem ID is a persistent identifier from a user's perspective. From the perspective of the chain, Diem IDs does not exist.
 
-# User story
+# End-to-End Experience
 Below is an example of using Diem ID domain for transferring money from one user to another. 
 
-* Receiver shares their Diem ID with the sender (outside the scope of this document)
-* Sender enters receiver's Diem ID and other relevant information into their wallet application
-* Sender's wallet attempts to fetch the receiver's profile picture and full name along with a confirmation dialog
-* Sender confirms the identity of the recipient
-* Sender presses ‘send’ button
-* Receiver's account is credited the amount in the transaction
-* Receiver's wallet displays recent transactions and their senders
+## Prerequisite:
+* VASPs get approval from association (via some offline process) on domain name
+```
+avasp, 0xf72589b71ff4f8d139674a3f7369c69b
+bvasp, 0xc5ab123458df0003415689adbb47326d
+```
+
+## User Story: 
+* Bob wants to receive funds from Alice
+* Bob registers a Diem ID at VASP B: `bob@bvasp`
+* Alice's Diem ID is registered at VASP A: `alice@avasp`
+* Bob shares either this Diem ID, `bob@bvasp` with Alice
+* Alice logs into VASP A, enters Bob’s identifier, an amount to pay, and submits payment
+* Alice’s VASP (VASP A) contacts Bob’s VASP’s (VASP B) off-chain API with the sender identifier `alice@avasp` and requests a reference_id, `rb1`
+* VASP A constructs a transaction with the specified amount and the reference_id `rb` and submits it to the diem network
+* VASP B receives a transaction with metadata containing `rb`, deposits the amount in Bob's account, and attaches the relevant metadata (e.g., Alice’s Diem ID) to the transaction so Bob can confirm the transaction
```

Comment:
Perhaps specify where the attachment happens (inside the VASP's internal DB, presumably?)